### PR TITLE
TP-1313 use partial when filtering by tag

### PR DESCRIPTION
**Pull request by aktfrikshun**
TP-1313 use partial when filtering by tag

### CODE REVIEW

File: app/controllers/admin/submissions_controller.rb [hunk 1]
```diff
@@ -36,12 +48,12 @@ def unflag
 
   def add_tag
     @submission.tag_list.add(admin_submission_params[:tag_list].split(","))
-    @submission.save
+    @submission.save!
```
Comment (ryanwoldatwork) [inline app/controllers/admin/submissions_controller.rb:51]: Why the bang here?

Comment (aktfrikshun) [reply app/controllers/admin/submissions_controller.rb:51]: To raise an exception if there is a problem saving the record.   I ran into an unusual case where I had several submissions and then made a field required on the 2nd page.   When I tagged submissions I had made prior to the required field change, the tags wouldn't be saved.  No error, nothing logged.   I then discovered the submission save was not completed due to the required field validation.   I added the bang so this condition would be raised.   

If this scenario seems likely in the future, we may want to handle that case more gracefully than throwing an exception.  I can write a ticket for that if so.

**Comment by aktfrikshun** [comment]
@ryanwoldatwork I added to this PR the isolating each report as a widget and loading async.  Check it out and let me know what you think.

File: app/controllers/admin/submissions_controller.rb (hunk 1 above)
Comment (ryanwoldatwork) [reply app/controllers/admin/submissions_controller.rb:51]: 👍

**Review by ryanwoldatwork — APPROVED** [review]

**Pull request merged**